prov/shm/src/smr_progress.c

```diff
@@ -917,7 +916,10 @@ static int smr_alloc_cmd_ctx(struct smr_ep *ep,
 		rx_entry->cq_data = cmd->msg.hdr.data;
 	}
 
-	if (cmd->msg.hdr.op_src == smr_src_inject) {
+	if (cmd->msg.hdr.op_src == smr_src_inline) {
+		memcpy(&cmd_ctx->cmd, cmd, sizeof(struct smr_msg_hdr) + cmd->msg.hdr.size);
```

Comment:
change this and below to `sizeof(cmd->msg.hdr)`

Reply:
done